PukingPredator/Assets/Scripts/Consumable.cs

```diff
@@ -159,8 +159,9 @@ public void PlaceAt(Vector3 position)
         if (state != ItemState.inInventory) { return; }
 
         gameObject.transform.localScale = initialScale;
-        //TODO: we should probably undo this at some point OR make it so you have control over the rotation
-        gameObject.transform.rotation = Quaternion.identity;
+
+        //TODO: we should probably remove this at some point OR make it so you have control over the rotation
+        //gameObject.transform.rotation = Quaternion.identity;
```

Comment:
I think its better to set the position to identity for now ideally we want to snap the rotation to some nice value. Right now, not resetting the rotation means the objects get a wierd rotation from the eating animation so puking the door for example will have it land on strange angles.

Reply:
I feel like puking things back at the rotation the objects were previously at makes the gameplay feel more intuitive, that's just my thoughts though

Reply:
Right, the issue right now is if you eat the door standing up the eating animation will cause it to tilt/fall and so if you spit it out it wont be standing it would be tilted due to the eating animation. So thats why i think having identity for now works better and then we can make the rotations snap to multiples of 90 for example so we dont get strange rotation from the eating animation.

Reply:
i haven't encountered it falling from the eating animation, it would do that if you were to bump into it before eating it, but the way that eating currently works shouldn't cause it to fall over